src/main/java/frc/robot/Robot.java

```diff
@@ -99,7 +102,7 @@ public void disabledPeriodic() {
   // This autonomous runs the autonomous command selected by your {@link RobotContainer} class. */
   @Override
   public void autonomousInit() {
-    AutonModes.setWaitTime(waitTimeSlider.getDouble(1));
+    AutonModes.setWaitTime(waitTimeSlider.getDouble(0));
```

Comment:
does it make sense to use the constant initialWaitTime here? If we change it again, one less place to have to remember to change it.

Reply:
That's a great idea. I'll make a constant called defaultInitialWaitTime.